assets/src/utils/charts.ts

```diff
@@ -3,19 +3,26 @@
 import uPlot, { AlignedData } from "uplot";
 import { getXTicksConfig } from "../utils/chartsXTicksConfig";
 
+const metricToTitle = new Map([
```

Comment:
Use simple js objects, no one reallly uses `Map` unless you have some performance issues and in this case you have none.